src/algorithm/branching.jl

```diff
@@ -174,6 +176,85 @@ function branching_score(
     return VariableBranchingScore(f⁻, f⁺, min(f⁻, f⁺))
 end
 
+function branching_score(
+    state::CurrentState,
+    form::DMIPFormulation,
+    bc::VariableBranchingCandidate,
+    parent_result::NodeResult,
+    parent_node::Node,
+    config::AlgorithmConfig{StrongBranching},
+)
+    sb_state = config.branching_rule.strong_branching_state
+    if !config.branching_rule.init
+        populate_base_model!(sb_state, form, parent_node, config)
+        config.branching_rule.init = true
+    elseif state.backtracking
+        reset_base_formulation_upon_backtracking!(sb_state, form, parent_node)
+    else
+        apply_branchings!(sb_state, parent_node)
+    end
+    sb_model = sb_state.gurobi_model
+
+    c = parent_result.cost
+    cvi = bc.index
+    xi = parent_result.x[index(cvi)]
+
+    low_bound = _approx_floor(xi, config.int_tol)
+    up_bound = _approx_ceil(xi, config.int_tol)
+    ci = state.constraint_state.base_state.var_constrs[index(cvi)]
+
+    c_up = _branch_cost!(sb_model, GT(up_bound), ci)
+    c_down = _branch_cost!(sb_model, LT(low_bound), ci)
+
+    μ = config.branching_rule.μ
+    f⁺ = c_down - c
+    f⁻ = c_up - c
+    f = (1 - μ)*min(f⁺, f⁻) + μ*max(f⁺, f⁻)
+    return VariableBranchingScore(f⁺, f⁻, f)
+end
+
+function _branch_cost!(
+    model::Gurobi.Optimizer,
+    constraint::LT,
+    ci::CI,
+)
+    interval = MOI.get(model, MOI.ConstraintSet(), ci)
+    temp_interval = IN(interval.lower, min(constraint.upper, interval.upper))
```

Comment:
You can get rid of the second method for `branch_cost!` by changing this logic to:
```suggestion
    temp_interval = IN(
        constraint isa LT ? interval.lower : max(constraint.lower, interval.lower),
        constraint isa GT ? min(constraint.upper, interval.upper) : interval.upper,
    )
```